File: Inheritance/Person.cpp
#include"Person.h"
Person::Person(){};
Person::Person(const char* nm,int d,int m,int y):name(nm),dob(d,m,y)
{
    // name=cstring(nm);
    // dob=Date(d,m,y);
}
void Person::display()
{
    name.display();
    cout<<endl;
    dob.display();
}

File: java/OOP3/Person.java
public class Person {
    String name;
    Date dob;
    public Person()
    {
        name=new String("NA");
        dob=new Date();
    }
    public Person(String name,int d,int m,int y)
    {
        this.name=name;
        dob=new Date(d,m,y);
    }
    public void display()
    {   
        System.out.println("Name: "+name);
        System.out.print("BirthDate: ");
        dob.showDate();
    }
}
